### ISSUE TITLE
Regression unable to use comments anymore in `turbo.json`

### ISSUE THREAD

**Issue by weyert** [vercel/turborepo#1283]
### What version of Turborepo are you using?

1.2.11

### What package manager are you using / does the bug impact?

pnpm

### What operating system are you using?

Mac

### Describe the Bug

In the past JSON5 support was added to `turbo.json` but adding `// comment` now breaks with:

```
error unmarshalling invalid character '/' looking for beginning of object key string
ERROR  turbo.json: invalid character '/' looking for beginning of object key string
```

### Expected Behavior

Able to use JSON5 format in `turbo.json` to allow comments

### To Reproduce

Add a comment e.g.

```
    "test": {
      // "dependsOn": ["^build"],
      "outputs": ["coverage/junit.xml"]
    },
```

**Comment by gsoltis** [comment]
Huh, it looks like this doesn't even work if I check out the [commit](https://github.com/vercel/turborepo/pull/745) that `json5` support was added in. Weirdly, comments in some other locations work. 
```jsonc
{
  "pipeline": {
    // This works
    "build": {
      "dependsOn": ["^build"],
      "outputs": ["dist/**", ".next/**"]
    }
  }
}
```
vs
```jsonc
{
  "pipeline": {
    "build": {
      // This doesn't work
      "dependsOn": ["^build"],
      "outputs": ["dist/**", ".next/**"]
    }
  }
}
```

I guess we need to move to a better `json5` library.

**Comment by weyert** [comment]
Oh no, that's a bummer. I would have sworn this was working

**Comment by bguedes-moz** [comment]
Just opened a [pull request](https://github.com/vercel/turborepo/pull/1472) or this issue.

Feel free to give feedback. :D

**Comment by weyert** [comment]
@bguedes-moz Awesome, thank you :))

**Comment by trevorr** [comment]
I'd also like to suggest adding to the documentation that JSON5 is supported. I didn't know about that until looking through the GitHub issues. Perhaps something [here](https://turborepo.org/docs/getting-started#create-turbojson) like "`turbo.json` supports [JSON5](https://json5.org/), so it can include comments."

**Comment by dobesv** [comment]
This is working for me today, maybe this can be closed?

**Comment by gsoltis** [comment]
Yup, fixed in #1472
